docs: add RN videos and discoverability improvements. (#18633)
supabase/supabase: apps/docs/pages/guides/auth/auth-magic-link.mdx

}
```

</TabPanel>
<TabPanel id="react-native" label="Expo React Native">

> Follow the [Expo Quickstart Guide](/docs/guides/getting-started/tutorials/with-expo-react-native#initialize-a-react-native-app) to corectly initialize `supabase-js` for usage with React Native.

Reviewer: ```suggestion
Follow the [Expo Quickstart Guide](/docs/guides/getting-started/tutorials/with-expo-react-native#initialize-a-react-native-app) to initialize `supabase-js` with React Native.
```

> When your user signs in, call [signInWithOtp()](/docs/reference/javascript/auth-signinwithotp) with their email address:

Reviewer: this is a bit confusing.

"when they sign in call signInWithOtp". So they have to sign in twice?

> When your user signs in, call [signInWithOtp()](/docs/reference/javascript/auth-signinwithotp) with their email address:

Author: ```suggestion
To start the sign in process call [signInWithOtp()](/docs/reference/javascript/auth-signinwithotp) with their email address:
```

```ts
import { makeRedirectUri } from "expo-auth-session";

const redirectTo = makeRedirectUri();

const { error } = await supabase.auth.signInWithOtp({
  email: "example@email.com",
  options: {
    emailRedirectTo: redirectTo,
  },
});
```

> For magic links you will need to handle deep linking into your application. Read the [docs](/docs/guides/auth/native-mobile-deep-linking).

Reviewer: ```suggestion
Read the [Deep Linking Documentation](/docs/guides/auth/native-mobile-deep-linking) to learn how to handle deep linking.
```
